akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -9,6 +9,7 @@ import scala.util.control.NonFatal
 import akka.stream.stage._
 import akka.stream.Supervision
 import akka.stream.impl.ReactiveStreamsCompliance
+import akka.stream.FlowMaterializer
```

Comment:
We have too many type parameters already: what else but a FlowMaterializer should it be?